spring-boot-admin-server/src/test/java/de/codecentric/boot/admin/notify/HipchatNotifierTest.java

```diff
@@ -0,0 +1,76 @@
+package de.codecentric.boot.admin.notify;
+
+import de.codecentric.boot.admin.event.ClientApplicationStatusChangedEvent;
+import de.codecentric.boot.admin.model.Application;
+import de.codecentric.boot.admin.model.StatusInfo;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.web.client.RestTemplate;
+
+import java.net.URI;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+/**
+ * @author Jamie Brown
+ */
+public class HipchatNotifierTest
+{
+    private HipchatNotifier notifier;
+    private RestTemplate restTemplate;
+
+    @Before
+    public void setUp()
+    {
+        restTemplate = mock(RestTemplate.class);
+        notifier = new HipchatNotifier();
+        notifier.setNotify(true);
+        notifier.setAuthToken("--token-");
+        notifier.setRoomId("-room-");
+        notifier.setUrl(URI.create("http://localhost/v2"));
+        notifier.setRestTemplate(restTemplate);
+    }
+
+    @Test
+    public void test_onApplicationEvent_resolve()
```

Comment:
Not sure if these comments are automatic, but the underscore convention was copied from the other Notifier Test classes. I wouldn't ordinarily use them myself unless implementing Cucumber tests, but figured this style was preferred.